lib/private/legacy/OC_Util.php

```diff
@@ -1100,7 +1100,7 @@ public static function getDefaultPageUrl() {
 			} else {
 				$appId = 'files';
 				$config = \OC::$server->getConfig();
-				$defaultApps = explode(',', $config->getSystemValue('defaultapp', 'files'));
+				$defaultApps = explode(',', $config->getSystemValue('defaultapp', 'dashboard'));
```

Comment:
```suggestion
				$defaultApps = explode(',', $config->getSystemValue('defaultapp', 'dashboard,files'));
```

Since dashboard can still be disabled and then we should still use files as a default app.